### PR TITLE
add -v and use standard version output

### PR DESCRIPTION
**Pull request by eikenb**
Bring in line with other consul-eco projects behavior.

Fixes: #125

### CODE REVIEW

**Review by lornasong — APPROVED** [review]
looks good to me! confirmed that the version outputted aligns with the issue description. left a non-blocking question

File: main.go [hunk 1]
```diff
@@ -48,8 +49,6 @@ func main() {
 
 	if isVersion {
 		fmt.Printf("%s\n", version.GetHumanVersion())
-		fmt.Printf("Compatible with Consul versions %s\n",
-			version.GetConsulVersionConstraint())
```
[the remark this answers is not in the document]
Comment (eikenb) [reply main.go:52]: Pretty much. They should probably know what version they need before downloading it and if they grab the wrong version I don't think having it in the version info really helps. Plus 1.4.1 is a lot older now.

What would probably help would be a nice error message if they try to run it with the wrong version. I think I'll add an issue to look into that option.

[the remark this answers is not in the document]
Comment (eikenb) [reply main.go:52]: Already has a nice error. So IMO we're good.

[the remark this answers is not in the document]
Comment (lornasong) [reply main.go:52]: Thanks for checking. That sounds good to me!

**Pull request merged**